roles/ispmail-certificate/tasks/pki.yml

```diff
@@ -0,0 +1,51 @@
+---
+- name: Create TLS directories
+  file:
+    state: directory
+    path: "{{ item }}"
+    owner: root
+    group: root
+    mode: 0770
+  loop:
+    - "{{ node_ca_key | dirname }}"
+    - "{{ node_ca_cert | dirname }}"
+  become: true
```

Comment:
Considering every task in the playbook has `become: true`, maybe this could be set at a playbook level.

Reply:
👍